app/helpers/schema_helper.py

```diff
@@ -64,4 +64,10 @@ def is_goto_rule(rule):
 
     @staticmethod
     def is_goto_meta_rule(rule):
```

Comment:
Doesn't goto_meta_rule imply that it's a goto rule containing a meta condition? Don't see the point in making this so wordy.